source/user-guide/environments-networks-json-explained.html.md.erb

```diff
@@ -0,0 +1,49 @@
+---
+owner_slack: "#modernisation-platform"
+title: `environments-networks` json explained
+last_reviewed_on: 2024-03-13
```

Comment:
Would be worth updating this one to todays date.

Reply:
Good catch! Should be fixed now :)